### ISSUE TITLE
shut down local pc without kubevpn leave deployments/xx, the next time kubevpn proxy deployments/xx (same header) won't work properly

### ISSUE THREAD

**Issue by kyr1eeee** [kubenetworks/kubevpn#114]
this is my situation:
1. I run this command:
`kubevpn proxy deployments/test-c3004 --headers name=kk`. 
And my local tun ip is 223.254.0.101, pod's req can be proxy to my local pc successfully.
2. I forgot to run command: 
`kubevpn leave deployments/test-c3004`.
And I shut down my pc.
3. I reboot my pc and run kubevpn command:
`kubevpn proxy deployments/test-c3004 --headers name=kk`.
it shows that my local tun ip turns to 223.254.0.103, and the ENVOY_CONFIG has two rules:
```yaml
ENVOY_CONFIG:
----
- Ports:
  - containerPort: 8003
    protocol: TCP
  Rules:
  - Headers:
      name: kk
    LocalTunIPv4: 223.254.0.101
    LocalTunIPv6: efff:ffff:ffff:ffff:ffff:ffff:ffff:999a
  - Headers:
      name: kk
    LocalTunIPv4: 223.254.0.103
    LocalTunIPv6: efff:ffff:ffff:ffff:ffff:ffff:ffff:999c
  Uid: deployments.apps.test-c3004
```
when i request for the pod with headers `name:kk`, it's a failure.
And i checked the envoy log, find out that envoy sidecar choose the "old cluster" 223.254.0.101:8003 which is not available.

So i wonder if the "pkg/handler/envoy.go:addEnvoyConfig" func could be optimized. when headers is same, rule shoule be updated instead of append?

**Comment by wencaiwulue** [comment]
Yes, it is a good idea to optimize it. it is so clear, i will fix it as soon as possible. Thank a lot ~

**Comment by wencaiwulue** [comment]
This issue is solved by commit 156ee998cd8d06d2aaf91249f3b4ced36e82ed4f

**Comment by kyr1eeee** [comment]
Yes, it's good now. So efficent!
